.travis.yml

```diff
@@ -23,8 +23,10 @@ before_install:
   - docker ps -a
 
 install:
-  - go get -u github.com/pilosa/go-pilosa
-  - cd "$GOPATH/src/github.com/pilosa/go-pilosa" && git checkout v0.9.0 && cd "$TRAVIS_BUILD_DIR"
+  - go get -u github.com/golang/dep/cmd/dep
+  - cd "$TRAVIS_BUILD_DIR"
+  - touch Gopkg.toml
+  - dep ensure -v -add "github.com/pilosa/go-pilosa@0.9.0" "github.com/pilosa/pilosa@master"
```

Comment:
maybe we could set the specific revision of pilosa instead, so we're in control if something changes

Reply:
Yup, they promised to release 1.0.2 y'day, but nothing happened, so I pulled master.
The plan is maybe to use `dep init` and set up certain version (1.0.2+,  because current 1.0.1 doesn't work for us).

Reply:
We can still pin a revision, even if it's not a release, right? I mean pinning it to the hash of current master